modules/services/mpd.nix

```diff
@@ -214,7 +210,18 @@ in {
       Install = { WantedBy = [ "sockets.target" ]; };
     };
 
+    launchd.agents.mpd = {
```

Comment:
```suggestion
    launchd.agents.mpd = lib.mkIf pkgs.stdenv.hostPlatform.isDarwin {
```

Reply:
Need a similar change for the systemd service

Reply:
that's actually a question I had. Is it really important? I think a systemd unit in Darwin isn't going to have much impact on the system. The opposite is probably also true. I don't mind adding the `if` condition, I'm just curious.

Reply:
I guess technically, we don't need to gate them within the module. But, we need to at least handle the test case to properly only expect a launchd config on darwin. 

```bash
building '/nix/store/fn0xc3v8nqwzbxavg844sfd1rxcsn5ql-nmt-report-mpd-launchd.drv'...
/nix/store/aqi31sid1n78h9afwbb6p9c57xpfh9r8-assertions.sh: line 31: /nix/store/c3vd1wdv6g6m95p17gda4byi6lbavd56-home-manager-generation/LaunchAgents/org.nix-community.home.mpd.plist: No such file or directory
tput: unknown terminal "xterm-kitty"
mpd-launchd: FAILED
Expected LaunchAgents/org.nix-community.home.mpd.plist to exist but it was not found.
```

Reply:
Thank you for your answer. I added the 2 `mkIf` expressions. Regarding the tests, I will wait to be able to run them either locally or on CI before doing anything.